docs/h2t-scc.md

```diff
@@ -0,0 +1,190 @@
+
+$ `oc edit scc <scc-name>`  
+Edit the SCC using this command. 
+You can edit SCC to define a set of conditions that a pod must run with in order to be accepted into the system.  
+FSGroup Strategy: RunAsAny
+
+
+## What is SCC?
+
+SCC is basically used for pod restriction, which means it defines the limitations for a pod, as in what actions it can perform and what all things it can access in the cluster.
+
+Advantages of using SCC.
+
+* Host directories can be used as volumes.
+* Able can set conditions for container user ID and Selinux Context of the container.
+* Able to run containers as privileged.
+* Able to control the use of host namespaces and networking.
+* Able to control usage of volume types.
+* Able to control capabilities that a container can request.
+* FSGroup can be allocated to pod volumes.
+
+OpenShift provides a set of predefined SCC that can be used, modified, and extended by the administrator.
+
+```
+$ oc get scc 
+NAME              PRIV   CAPS  HOSTDIR  SELINUX    RUNASUSER         FSGROUP   SUPGROUP  PRIORITY
+anyuid            false   []   false    MustRunAs  RunAsAny          RunAsAny  RunAsAny  
+hostaccess        false   []   true     MustRunAs  MustRunAsRange    RunAsAny  RunAsAny  
+hostmount-anyuid  false   []   true     MustRunAs  RunAsAny          RunAsAny  RunAsAny
+nonroot           false   []   false    MustRunAs  MustRunAsNonRoot  RunAsAny  RunAsAny 
+privileged        true    []   true     RunAsAny   RunAsAny          RunAsAny  RunAsAny
+restricted        false   []   false    MustRunAs  MustRunAsRange    RunAsAny  RunAsAny
+```
+If one wishes to use any pre-defined scc, that can be done by simply adding the user or the group to the scc group.
+
+$ `oadm policy add-user-to-scc <scc_name> <user_name>`
+
+$ `oadm policy add-group-to-scc <scc_name> <group_name>`
+
+OpenShift guarantees that the capabilities required by a container are granted to the user that executes the container at admission time.  We can not allow any container to get access to unnecessary capabilities or to run in an insecure way (e.g. privileged or as root)
+
+Adding a regular user or to a group given access to the SCC allows them to run privileged pods. There are mainly four section in SCC to control access to volumes in OpenShift.
+
+1. Supplemental Groups
+1. fsGroup
+1. runAsUser
+1. seLinuxOptions
+
+### 1)Supplemental Groups
+Supplemental groups are regular Linux groups. 
+When a process runs in the system, it runs with a user ID and group ID. 
+These groups are used for controlling access to shared storage.
+
+* Check the NFS mount using the following command.
+
+   `# showmount -e <nfs-server-ip-or-hostname>`<br />
+   Export list for f21-nfs.vm: <br />
+   /opt/nfs * 
+
+* Check NFS details on the mount server using the following command.
+   
+   `# cat /etc/exports` <br />
+   /opt/nfs *(rw,sync,no_root_squash)
+
+*  `# ls -lZ /opt/nfs -d` <br />
+   drwxrws---. nfsnobody 2325 unconfined_u:object_r:usr_t:s0 /opt/nfs
```

Comment:
```suggestion
* Check owner of exported directory

    ```
     $ ls -lZ /opt/nfs -d
     drwxrws---. nfsnobody 2325 unconfined_u:object_r:usr_t:s0 /opt/nfs
    ```
```

Same idea with a leading text at the bullet...